contracts/abi.package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@origin/defi",
+  "license": "MIT",
+  "homepage": "https://github.com/originprotocol/origin-dollar",
+  "engines": {
+    "node": "16"
+  }
```

Comment:
this file is missing a version, and it's probably better to automate the versioning here

Reply:
Will look into ways to do it